test/hotspot/jtreg/compiler/vectorization/TestEor3AArch64.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2022, Arm Limited. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package compiler.vectorization;
+
+import jdk.test.lib.Asserts;
+import compiler.lib.ir_framework.*;
+
+import java.util.Random;
+
+import jdk.test.lib.Utils;
+
+/*
+ * @test
+ * @bug 8293488
+ * @summary Test EOR3 Neon/SVE2 instruction for aarch64 SHA3 extension
+ * @library /test/lib /
+ * @requires os.arch == "aarch64" & vm.cpu.features ~=".*sha3.*"
```

Comment:
```suggestion
 * @requires os.arch == "aarch64" & vm.cpu.features ~= ".*sha3.*"
```
nit: [style] it's better to have one extra space.

Reply:
@shqking  Thank you for the comments. I have made the suggested changes.